index.js

```diff
@@ -135,16 +136,27 @@ module.exports = function(source) {
                         });
                     });
 
-                    node.update(`(${
-                        // Each tech has own generator
-                        Object.keys(techToFiles)
-                            // js tech is always last
-                            .sort(a => extToTech[a] === 'js')
-                            .map(tech =>
-                                (generators[extToTech[tech] || tech] || generators['*'])(techToFiles[tech])
-                            )
-                            .join(',\n')
-                    })`);
+                    let defaultTech;
+
+                    // Each tech has own generator
+                    const res = Object.keys(extToFiles)
+                        // use techs from config for order
+                        // so the first one would be default, `js` in most cases
+                        .sort((a, b) => techs.indexOf(extToTech[a]) - techs.indexOf(extToTech[b]))
+                        .map((ext, i) => {
+                            const tech = extToTech[ext] || ext;
+                            i || (defaultTech = tech);
```

Comment:
and here.

But they are both not file extensions, they both can contain a dot (`.`) inside (like `react.js`).

Reply:
So how to name them? any ideas? `extWithSuffix`? or just `suffix`?